dashboard/app/controllers/curriculum_proxy_controller.rb

```diff
@@ -0,0 +1,38 @@
+require 'net/http'
+require 'set'
+
+# This proxy controller allows us to serve curriculumn builder docs that live on
+# [docs|curriculum].code.org as if they live on studio.code.org. This is done so that we can
+# render them in an iframe, and access that iframe without having cross-origin problems
+class CurriculumProxyController < ApplicationController
+  include ProxyHelper
+
+  EXPIRY_TIME = 30.minutes
+
+  def get
+    full_route = URI.parse(request.original_url).path
+    if params[:docs_route]
```

Comment:
If I'm reading this correctly, one or the other of `docs_route` or `curriculum_route` params must be truthy but their actual values are irrelevant and the proxy route comes from the request url. In that case, do we need those params? How about instead just parsing the request route and branching on the first path part?